installer/manifests/keptn/charts/control-plane/values.yaml

```diff
@@ -146,6 +146,8 @@ shipyardController:
   config:
     taskStartedWaitDuration: "10m"
     uniformIntegrationTTL: "48h"
+    disableLeaderElection: true
+    replicas: 3
```

Comment:
I think the number of replicas should stay at 1 per default until https://github.com/keptn/keptn/pull/6977 is merged

Reply:
because of [this](https://github.com/keptn/keptn/pull/6967/files#diff-4287874700cb1d1f460fd185db5a5bed707a6c5868649ebe663b5380dbbb765aR19) replicas are set to 1 anyway, do you think it is better still to explicitly make them 1 there too?

Reply:
maybe i did not understand it correctly but isn't disabling LE and setting replicas to more than 1 kind of contradicting?

Reply:
yeah it can be confusing for the user but it works :D I will set it to 1 for now so that it does not sounds strange